src/contracts/mod.rs

```diff
@@ -0,0 +1,29 @@
+use ethers::prelude::*;
+
+abigen!(
+    AggregatedWallet,
+    "$OUT_DIR/IAggregatedWallet.sol/IAggregatedWallet.json"
+);
+
+abigen!(Aggregator, "$OUT_DIR/IAggregator.sol/IAggregator.json");
+
+abigen!(
+    Create2Deployer,
+    "$OUT_DIR/ICreate2Deployer.sol/ICreate2Deployer.json"
+);
+
+abigen!(EntryPoint, "$OUT_DIR/IEntryPoint.sol/IEntryPoint.json");
+
+abigen!(Paymaster, "$OUT_DIR/IPaymaster.sol/IPaymaster.json");
+
+abigen!(
+    StakeManager,
+    "$OUT_DIR/IStakeManager.sol/IStakeManager.json"
+);
+
+abigen!(Wallet, "$OUT_DIR/IWallet.sol/IWallet.json");
+
+abigen!(
+    UserOperation,
+    "$OUT_DIR/UserOperation.sol/UserOperationLib.json"
```

Comment:
@Vid201 Just FYI. Here it would generate the same data structure `UserOperation`.

Reply:
Hmm, ok. Then we can import this data structure in `user_operation.rs` (and implement additional things that will be needed).